src/potions.ts

```diff
@@ -476,10 +477,26 @@ export const rufusPotion = new Potion($item`closed-circuit pay phone`, {
 
     // We consider the average price of the shadow items to not get gated behind an expensive one
     const shadowItems = $items`shadow brick, shadow ice, shadow sinew, shadow glass, shadow stick, shadow skin, shadow flame, shadow fluid, shadow sausage, shadow bread, shadow venom, shadow nectar`;
-    const averagePrice =
-      sum(shadowItems, (it) =>
-        historical && historicalAge(it) < 14 ? historicalPrice(it) : mallPrice(it)
-      ) / shadowItems.length;
+    const getPrice = (item: Item): number =>
+      3 * (historical && historicalAge(item) < 14 ? historicalPrice(item) : mallPrice(item));
+    const avgSpeculativePrice = sum(shadowItems, getPrice) / shadowItems.length;
+
+    if (haveItemQuest) return avgSpeculativePrice;
+
+    if (get("rufusDesiredItems") === "") {
+      withChoice(1497, 6, () => use($item`closed-circuit pay phone`));
+      if (handlingChoice()) {
+        throw `Stuck in the closed-circuit pay phone dialogue? Please manually hang up and re-run`;
```

Comment:
I think we generally prefer to throw Error objects instead of strings, although I'm not entirely if there's a compelling reason for that convention.